README.md

```diff
@@ -204,6 +204,7 @@ and the following linters are disabled by default:
 $ golangci-lint help linters
 ...
 Disabled by default linters:
+bodyclose: checks whether HTTP response body is closed successfully [fast: false, auto-fix: false]
```

Comment:
I would include this change in another PR

Reply:
Fixed.